src/frontend/github-oauth.ts

```diff
@@ -91,12 +89,23 @@ export default (opts: OAuthOptions): GithubOAuth => {
       );
 
       req.session.githubToken = response.data.access_token;
+
       if (!req.session.githubToken) {
         return next(new Error('Missing Access Token from Github OAuth Flow.'));
       }
-      return res.redirect(redirectUrl);
+
+      return res.redirect(redirectUrl.toString());
```

Comment:
It's already a string.

Reply:
I was getting an error that it is  `UNKNOWN`

Reply:
really?  Huh.  Still, you have the `|| ""` up above which will make it a string

Reply:
That's what I thought but I see this when I remove `toString()`

<img width="759" alt="Screen Shot 2021-06-24 at 7 00 46 pm" src="https://user-images.githubusercontent.com/37155488/123234699-81e5fa80-d51e-11eb-81c0-0a2b5737e8f1.png">

Reply:
I can have just `redirectUrl.toString()` and omit the `|| ''` Error goes away with that.

Reply:
wait.  Is this an error only showing up in your IDE?  Does it pass with when you do `npm run build`?

Reply:
Hm, might be something wrong with your cookie session typings?  Either way, my current version returns `any` which isn't good anyways.  please change the line to `const redirectUrl = req.session[state] as string;` and it should work.

Reply:
![Screen Shot 2021-06-28 at 12 46 13 pm](https://user-images.githubusercontent.com/37155488/123572075-dafbaa00-d80e-11eb-8fbb-c2f47ec7c057.png)

Git blame says it's your typing 😛  but I can change it.

Reply:
well, my IDE is saying that it's `any` and is fine without errors, so I think this is a vscode issue :)

Reply:
You IDE-ist.

Reply:
:P